<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QF4CPlus_VS_QtStateMachineClass</class>
 <widget class="QMainWindow" name="QF4CPlus_VS_QtStateMachineClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QF4CPlus_VS_QtStateMachine</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="push_QF4Start">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>111</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>QF4CPlus_Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="push_QTStart">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>150</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>QT_HSM_Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="qFCount">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="QTCount">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="QFAllTime">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>320</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="QTAlltime">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QF4CPlus_VS_QtStateMachine.qrc"/>
 </resources>
 <connections/>
</ui>
